scripts/broker.csx

```diff
@@ -20,32 +20,29 @@ public void AddUserToVirtualHost() => CreateUserPermissionRequest("PUT", GetBrok
 
 public Broker GetBroker()
 {
-    var connectionStringBuilder = new DbConnectionStringBuilder
+    var connectionString = Environment.GetEnvironmentVariable("RabbitMQTransport.ConnectionString");
+
+    if (string.IsNullOrEmpty(connectionString))
     {
-        ConnectionString = Environment.GetEnvironmentVariable("RabbitMQTransport.ConnectionString")
-    };
+        throw new Exception("The 'RabbitMQTransport.ConnectionString' environment variable is not set.");
+    }
 
-    ApplyDefault(connectionStringBuilder, "username", "guest");
-    ApplyDefault(connectionStringBuilder, "password", "guest");
-    ApplyDefault(connectionStringBuilder, "virtualhost", "nsb-rabbitmq-test");
-    ApplyDefault(connectionStringBuilder, "host", "localhost");
+    var connectionStringBuilder = new DbConnectionStringBuilder { ConnectionString = connectionString };
 
     return new Broker
     {
-        UserName = connectionStringBuilder["username"].ToString(),
-        Password = connectionStringBuilder["password"].ToString(),
-        VirtualHost = connectionStringBuilder["virtualhost"].ToString(),
-        HostName = connectionStringBuilder["host"].ToString(),
+        UserName = connectionStringBuilder.GetOrDefault("username", "guest"),
+        Password = connectionStringBuilder.GetOrDefault("password", "guest"),
+        VirtualHost = connectionStringBuilder.GetOrDefault("virtualhost", "/"),
+        HostName = connectionStringBuilder.GetOrDefault("host", "localhost"),
```

Comment:
As I commented on the Transport tests, not sure we should be using localhost as a default here, since we don't do that in `ConnectionConfiguration`.